### PR TITLE
Add status updates sidekiq job

### PR DESCRIPTION
**Pull request by deborahchua**
We are moving away from using a healthcheck to monitor the status
of delivery attempts created within the past hour.

This sidekiq job will run every 1 min and send the total number
of delivery attempts as well as the number of pending to Graphite.
We can use StatD's `gauge` feature to maintain the current values
until they are next updated.

Trello card: https://trello.com/c/hPiOuVko/1581-5-extract-the-statusupdates-check-from-the-healthcheck-endpoint-in-email-alert-api-to-a-separate-icinga-check

### CODE REVIEW

File: app/services/metrics_service.rb [hunk 1]
```diff
@@ -77,5 +77,17 @@ def time(metric, &block)
     def timing(namespace, difference)
       GovukStatsd.timing(namespace, difference)
     end
+
+    def gauge(stat, metric)
+      statsd.gauge(stat, metric)
+    end
+
+    def statsd
```
Comment (cbaines) [inline app/services/metrics_service.rb]: It would be good to name this in some way to identify why it should be used compared to the `GovukStatsd` client.

Comment (deborahchua) [reply app/services/metrics_service.rb]: Thanks @cbaines , I have updated this in https://github.com/alphagov/email-alert-api/pull/1073/commits/6c284698227eb0b3f380f036e4839051beee0382

File: app/services/metrics_service.rb [hunk 2]
```diff
@@ -64,6 +66,14 @@ def delivery_attempt_status_changed(status)
       increment("delivery_attempt.status.#{status}")
     end
 
+    def delivery_attempt_pending_status_total(total)
+      OverallMetricsService.gauge("delivery_attempt.pending_status_total", total)
+    end
+
+    def delivery_attempt_total(total)
+      OverallMetricsService.gauge("delivery_attempt.total", total)
+    end
```
Comment (thomasleese) [inline app/services/metrics_service.rb]: I wonder if it's better to make these methods of `OverallMetricsService` to make the distinction explicit.

File: app/services/overall_metrics_service.rb [hunk 3]
```diff
@@ -0,0 +1,13 @@
+module OverallMetricsService
```
Comment (thomasleese) [inline app/services/overall_metrics_service.rb]:
```suggestion
module GlobalMetricsService
```

I wonder if this is a clearer name for this service.

File: app/services/global_metrics_service.rb [hunk 4]
```diff
@@ -0,0 +1,25 @@
+class GlobalMetricsService
+  class << self
+    def statsd
```
Comment (thomasleese) [inline app/services/global_metrics_service.rb]: Can this also be private?

Comment (deborahchua) [reply app/services/global_metrics_service.rb]: Thanks Tom! I have updated this now

**Pull request merged**